charms/worker/k8s/charmcraft.yaml

```diff
@@ -98,7 +106,16 @@ actions:
 
 parts:
   charm:
-    build-packages: [git]
+    charm-binary-python-packages:
+      - setuptools
+      - cryptography
+    build-packages:
+      - git
+      - libffi-dev
+      - libssl-dev
+      - pkg-config
+      - rustc
+      - cargo
```

Comment:
Is there some chance these packages which are used by the focal build machine create something that doesn't work in jammy/noble.  Hopefully not.  I know our integration tests only run on focal ATM

Perhaps we need a matrix of testing against jammy/noble in the future

Reply:
Sure, I'll create a card to track this process